src/test/java/benchmark/OverlappingFieldValidationBenchmark.java

```diff
@@ -75,8 +76,15 @@ private String readFromClasspath(String file) throws IOException {
     }
 
     @Benchmark
-    public List<ValidationError> overlappingFieldValidation(MyState myState) {
-        return validateQuery(myState.schema, myState.document);
+    public void overlappingFieldValidationAbgTime(MyState myState, Blackhole blackhole) {
+        blackhole.consume(validateQuery(myState.schema, myState.document));
+    }
+
+    @Benchmark
+    @BenchmarkMode(Mode.Throughput)
+    @OutputTimeUnit(TimeUnit.SECONDS)
+    public void overlappingFieldValidationThroughput(MyState myState, Blackhole blackhole) {
+        blackhole.consume(validateQuery(myState.schema, myState.document));
```

Comment:
Added throughput

I tried to find some perf improvements in Overlapping fields code but none to be easily found